tempto-core/src/main/java/io/trino/tempto/assertions/QueryAssert.java

```diff
@@ -22,6 +22,7 @@
 import io.trino.tempto.query.QueryExecutor;
 import io.trino.tempto.query.QueryResult;
 import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.AbstractThrowableAssert;
```

Comment:
first to commits already in https://github.com/trinodb/tempto/pull/76

Reply:
was intentional to avoid conflcts. rebased now, with no other changes